db/migrate/20180710094156_devise_create_users.rb

```diff
@@ -22,10 +22,10 @@ def change
       t.inet     :last_sign_in_ip
 
       ## Confirmable
-      # t.string   :confirmation_token
-      # t.datetime :confirmed_at
-      # t.datetime :confirmation_sent_at
-      # t.string   :unconfirmed_email # Only if using reconfirmable
+      t.string   :confirmation_token
```

Comment:
you can't update existing migrations.
If you need to change something you need to create new migration

Reply:
https://edgeguides.rubyonrails.org/active_record_migrations.html#changing-existing-migrations